Checkbox/Checkbox.html

```diff
@@ -1,4 +1,10 @@
-<template class="d-reset d-inline" role="presentation">
-	<input type="checkbox" role="checkbox" attach-point="focusNode,valueNode"
-		checked="{{checked}}" value="{{value}}" name="{{name}}"/>
+<template role="presentation">
+
+	<input class="d-reset" type="checkbox" attach-point="focusNode,valueNode"	
```

Comment:
`d-reset` set font, line-height and color. This looks not needed here if only padding and margin must be set. Same comment for radiobutton.